lib/modules/dosomething/dosomething_signup/dosomething_signup.install

```diff
@@ -64,6 +64,12 @@ function dosomething_signup_schema() {
         'length' => 255,
         'not null' => FALSE,
       ),
+      'competition' => array(
+        'description' => 'Boolean indicating the use signed up to a competition',
+        'type' => 'int',
+        'not null' => TRUE,
```

Comment:
Don't we want it to be `null` if the user never encountered the competitions form for campaigns? `false` if they don't want to participate, `true` if they do participate, and `null` if they were never asked yet cause they haven't gone a competitions available campaign?